### PR TITLE
add authentication entry to support table

### PR DESCRIPTION
**Pull request by SimoneDutto**

### CODE REVIEW

**Review by alesstimec — DISMISSED** [review]

File: reference/rebac_admin_backend.rst [hunk 1]
```diff
@@ -23,13 +23,14 @@ JAAS Implementation
...
    ✅   ``entitlements``     
    ✅   ``capabilities`` 
    ✅   ``groups``     
    ✅   ``resources``     
-   🟡   ``identities``    no support for creation, update and deletion.
-   ❌   ``roles``         no support for roles.
- ====== ================ =========================================================================
+   🟡   ``identities``     no support for creation, update and deletion.
+   ❌   ``roles``          no support for roles.
+   ❌   ``authentication`` no support for authentication.
```
Comment (babakks) [inline reference/rebac_admin_backend.rst]: Maybe a bit more clear like: "no support for authenticating providers".

**Review by babakks — DISMISSED** [review]

File: reference/rebac_admin_backend.rst (hunk 1 above)
Comment (SimoneDutto) [reply reference/rebac_admin_backend.rst]: did it

File: reference/rebac_admin_backend.rst [hunk 2]
```diff
@@ -23,13 +23,14 @@ JAAS Implementation
...
    ✅   ``entitlements``     
    ✅   ``capabilities`` 
    ✅   ``groups``     
    ✅   ``resources``     
-   🟡   ``identities``    no support for creation, update and deletion.
-   ❌   ``roles``         no support for roles.
- ====== ================ =========================================================================
+   🟡   ``identities``     no support for creation, update and deletion.
+   ❌   ``roles``          no support for roles.
+   ❌   ``authentication`` no support for authentication providers.
```
Comment (kian99) [inline reference/rebac_admin_backend.rst]: What are authentication providers in the rebac world btw?

**Review by kian99 — DISMISSED** [review]

File: reference/rebac_admin_backend.rst (hunk 2 above)
Comment (SimoneDutto) [reply reference/rebac_admin_backend.rst]: image

the swagger is not public yet, i paste here the screenshot to provide more context

Comment (alesstimec) [reply reference/rebac_admin_backend.rst]: Maybe, instead of `authentication`, we could say `no support for identity provider management`

**Review by alesstimec — DISMISSED** [review]
LGTM, but see my suggestion for rewording

**Review by ale8k — DISMISSED** [review]

File: reference/rebac_admin_backend.rst (hunk 2 above)
Comment (babakks) [reply reference/rebac_admin_backend.rst]: Yeah, I agree with @alesstimec's suggestion.

Comment (babakks) [reply reference/rebac_admin_backend.rst]: @kian99 It's more about the supported IdP (those that identities can be retrieved from and added to). And I agree that this is a bit unrelated to ReBAC stuff, but that's already part of the OpenAPI spec.

**Review by kian99 — APPROVED** [review]

**Review by babakks — APPROVED** [review]

**Pull request merged**